### DISCUSSION TITLE
Email

### DISCUSSION THREAD

**Discussion by g105b** [SuperHyperInstantFutureTime/TrackShift#77] [General]
We're going to need email setting up as soon as we have customers. The usual go-to is Gmail, which I personally have a lot of experience in as my business has been a reseller of their enterprise software for over 10 years. However, they've consistently proven to be terrible at customer support, and I'm currently right in the middle of a simple support issue that's making my blood boil.

Putting my current issues aside, Google definitely do not share the same values as Super Hyper Instant Future Time do, so it's well worth looking for alternatives, and we can track our discussion here.

At first I thought we could go with any other provider, but there are actually a few requirements that need to be met, which I know are not met by certain Google alternatives out there.

Here's my simple list of requirements:

- Email must not be bound to one platform. Changing MX records on a domain should be all it takes to switch elsewhere.
- Standard IMAP/SMTP, so we can add the mailbox to any software we like.
- Calendar - it seems simple, but a lot of providers don't offer any ability to work with calendar invites, or if they do, they mess it up.
- Contacts - even simpler than calendar, I don't want to maintain a list of contacts elsewhere.
- Security - at least basic security like SPF, so we have half a chance at staying out of people's spam folders. I don't really care that much about encryption and the like, because unless the receiver encrypts their email too, it has to be sent in plain text anyway.

In short, it should work just like Gmail does, but preferably with less stress when things go wrong.

Initial list of alternatives that I've heard of that we can evaluate:

- Tutanota - https://tutanota.com/
- Proton - https://proton.me/
- Fast Mail - https://www.fastmail.com/
- Office 365 - https://www.microsoft.com/en-us/microsoft-365/outlook/outlook-for-business
- Zoho - https://www.zoho.com/en-uk/
- Greatmail - https://www.greatmail.com/

@richardbirkin please add to this list if you know of or can find any more to evaluate.

**Comment by richardbirkin** [comment]
### Tutanota 
Seems closest in terms of shared values - privacy, open-source, sustainable, ad-free. They also have calendar. Support looks a little light, and it seems they disable accounts that break T&Cs, which shouldn't be a problem unless it is through no fault of our own. 

### Zoho
Zoho is ok - I've used it for clients before. It's enterprise level which might be a bit big for us at first. They do have a CRM which would come in handy for growth. 

### Proton
A really nice offering. Again, focussed on privacy and freedom. Has all the functionality we're looking for.
Also invented at CERN, which is (if nothing else) CAF. Pricey though. 

### Office 365
I'm just going to say no to this. It's the opposite of everything we stand for. 

### Greatmail
Seems standard enough. USA based though which means support could be out of sync with us. 

### Fast Mail 
Cheaper than Proton but just as focussed on privacy. Has calendar. 


# Conclusion

My shortlist from quick analysis:

- Fast Mail
- Tutanota
- Proton

**Reply by g105b** [reply]
I can discount Proton - they've made an unorthodox decision of disallowing email clients. Therefore, you must use their web interface for email and calendar, rather than the email client of your operating system. On top of that, there are quite a few annoying bugs with calendar. A client of mine uses it and I can confirm the bugs are annoying. Each time a meeting invite is updated, it has to create a new meeting over the top of the old one with the word CANCELLED in the title, then a new one for the updated meeting. It's really messy and doesn't feel right.

I've had a personal recommendation of Fast Mail, so we probably should give that a go. I can't say anything for Tutanota.

**Reply by richardbirkin** [reply]
I think we go with Fast Mail then. 
Good decision making. 

As soon as the Innovate UK funding lands I'll setup an account.
